SCClassLibrary/Common/GUI/PlusGUI/Math/PlotView.sc

```diff
@@ -977,6 +975,30 @@ Plotter {
 		};
 	}
 
+	// Convenience methods to dispatch settings shared between all plot grids
+	tickLabelOffset_{ |ax, pnt| this.prSetInSubplotGrid(ax, 'labelOffset_', pnt) }
+	tickLabelOffset { |ax| ^this.prGetInSubplotGrid(ax, 'labelOffset') }
+	tickLabelAlign_{ |ax, sym| this.prSetInSubplotGrid(ax, 'labelAlign_', sym) }
+	tickLabelAlign { |ax| ^this.prGetInSubplotGrid(ax, 'labelAlign') }
+	tickLabelAnchor_{ |ax, sym| this.prSetInSubplotGrid(ax, 'labelAnchor_', sym) }
+	tickLabelAnchor { |ax| ^this.prGetInSubplotGrid(ax, 'labelAnchor') }
+	tickLabelAppendString_{ |ax, str| this.prSetInSubplotGrid(ax, 'labelAppendString_', str) }
+	tickLabelAppendString { |ax| ^this.prGetInSubplotGrid(ax, 'labelAppendString') }
+	constrainTickLabelExtents_{ |ax, bool| this.prSetInSubplotGrid(ax, 'constrainLabelExtents_', bool) }
+	constrainTickLabelExtents { |ax| ^this.prGetInSubplotGrid(ax, 'constrainLabelExtents') }
+
+	prSetInSubplotGrid { |axis, method, val|
+		plots.do { |plot|
+			plot.drawGrid.perform(axis.asSymbol).perform(method, val)
+		};
+		this.updatePlotBounds;
+	}
+	prGetInSubplotGrid { |axis, method, val|
+		// assume all plot grids have the same properties
+		// (this is an convenience anyway)
+		^plots[0].drawGrid.perform(axis).perform(method);
+	}
```

Comment:
Would like feedback on this implementation:
These are convenience methods to set tick label properties of all of the subplots. 
In order to make the "setters", they also had to be defined as instance variables of `Plotter`. 
Because there are 2 axes with ticks (`x` and `y`), rather than making twice as many methods, I made the axis a variable. As a result the "getters" take an `ax` argument. Getters that take arguments seems odd, but perhaps ok in this scenario?

Reply:
I don't want to be the final adjudicator on these methods, but I'd lean towards either duplicating methods ( `.tickLabelOffsetX`/`.tickLabelOffsetY`) or using array of arguments, OR two separate args ( `.tickLabelOffsets_(pntX, pntY)` or `.tickLabelOffsets_([pntX, pntY])`), with the ability to provide nil as either argument when we want to set only one of them.

Reply:
- setters can/should only have one single argument (otherwise you can't write `something.label = "hello"`).
- is there a reason why to call `labelAlign` instead `tickLabelAlign` when it is called on the subplots? 
- setters like `labelOffsetX`, as Marcin suggests
- instead of wrapping everything in private methods, it might be better to have a method that gives you: `plot[0].drawGrid.perform(axis)`

e.g.
```supercollider
representativeGrid { |axis|
^plot[0].drawGrid.perform(axis)
}
```

then you can write:

```supercollider
labelOffsetX_ { |point| this.representativeGrid(\x).labelOffset_(point) }
```

A note:
Writing `asSymbol`, like in  `plot.drawGrid.perform(axis.asSymbol)` over time has proven to be a bad idea in some cases. You make code work that shouldn't really.

Reply:
Thanks @dyfer and @telephon for your insightful comments as usual :) I've reworked things based on your feedback and it's getting closer.

>is there a reason why to call `labelAlign` instead `tickLabelAlign` when it is called on the subplots?

(Thanks for asking about this, it made me look more closely at anatomy of `Plotter`'s components again...)
Yes, `Plot` had a concept of axis labels, just called `labelX/Y` but not tick labels. 
`DrawGridX/Y` had a concept of tick labels, which it calls "label" (e.g. `labelOffset`), but no concept of axis labels. 
At the level of `Plotter` we distinguish between the two. So when Plotter acts on it's axis labels, it now sets `axisLabelX/Y` (used to just be `labelX/Y`), and when it acts on the grid labels, it calls them `tickLabelParamX/Y` (e.g. `tickLabelAlignX`). Make sense?

A long term goal is to do 90% of what you might need through `Plotter`, so you rarely have to go into individual plots/grids, where conventions and responsibility become murky. 

>instead of wrapping everything in private methods, it might be better to have a method that gives you: `plot[0].drawGrid.perform(axis)`

Curious what you think of the new approach, which ditches private methods for more useful ones: b17c7b429a4cba7394e313de639ba7c92a7a6d5e

Thanks for being patient with this... there are quite a lot of details to manage, but approaching a milestone that can be suitable for the RC.